db/migrate/20160712045614_create_tasks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTasks < ActiveRecord::Migration
+  def change
+    create_table :tasks do |t|
```

Comment:
The `projects` table is using `uuid` for the primary key. Not technically necessary on `tasks` but it would probably be good for consistency. You can see the format of the declaration on the other migration if you aren't familiar with it.